### PR TITLE
compact protocol/log layout

### PR DESCRIPTION
**Pull request by hannesa2**
It needs https://github.com/owncloud/android-library/pull/225

(cherry picked from commit 046948525838ad1a575110afc1683f393c28a621)

This is the layout improvement to `compact logfile` https://github.com/owncloud/android-library/pull/225

owncloudlog 2

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/ui/activity/LogHistoryActivity.java [hunk 1]
```diff
@@ -77,7 +70,7 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.logs);
         setupToolbar();
 
-        mLayoutManager = new LinearLayoutManager(this);
+        RecyclerView.LayoutManager mLayoutManager = new LinearLayoutManager(this);
```
Comment (davigonz) [inline owncloudApp/src/main/java/com/owncloud/android/ui/activity/LogHistoryActivity.java]: Rename `mLayoutManager` to `layoutManager`

File: owncloudApp/src/main/res/layout/log_item.xml [hunk 2]
```diff
@@ -16,24 +15,10 @@
...
-
-</LinearLayout>
\ No newline at end of file
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/logContent"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:lines="1"
+    tools:text="hola" />
```
Comment (davigonz) [inline owncloudApp/src/main/res/layout/log_item.xml]: Please delete this line and `xmlns:tools="http://schemas.android.com/tools"` since are not needed.

**Comment by davigonz** [comment]
Just a couple of tiny changes @hannesa2 and we are ready to test @jesmrec

File: owncloudApp/src/main/res/layout/log_item.xml (hunk 2 above)
Comment (hannesa2) [reply owncloudApp/src/main/res/layout/log_item.xml]: It's used !
`tools:text="hola"`

**Comment by jesmrec** [comment]
Logs do not have margin inside the layout. A little margin will make the text nicer to read.

screen shot 2019-01-09 at 15 42 05

screen shot 2019-01-09 at 15 42 20

**Comment by jesmrec** [comment]
@hannesa2 what do you think about my comment https://github.com/owncloud/android/pull/2415#issuecomment-452720386?

**Comment by hannesa2** [comment]
Of course, here it is
image

**Comment by hannesa2** [comment]
But **much more important** is to merge this https://github.com/owncloud/android-library/pull/225 (and of course the [others](https://github.com/owncloud/android-library/pulls) too.) otherwise you see nothing

**Pull request merged**